### PR TITLE
[daint dom] malt

### PR DESCRIPTION
**Pull request by jgphpc**
https://jira.cscs.ch/browse/UES-166

### CODE REVIEW

File: jenkins-builds/6.0.UP07-18.08-gpu [hunk 1]
```diff
@@ -81,6 +81,7 @@
  jengafettplugin-0.1.eb                             --set-default-module --installpath=$APPS/UES/jenkins/6.0.UP07/gpu/easybuild/tools
  likwid-4.3.3-perf_event.eb                         --set-default-module --installpath=$APPS/UES/jenkins/6.0.UP07/gpu/easybuild/tools
  line_profiler-2.1-CrayGNU-18.08-python3.eb         --set-default-module --installpath=$APPS/UES/jenkins/6.0.UP07/gpu/easybuild/tools
+ malt-1.0.0-CrayGNU-18.08.eb                        --set-default-module --installpath=`echo $APPS`/UES/jenkins/6.0.UP07/gpu/easybuild/tools
```
Comment (kraushm) [inline jenkins-builds/6.0.UP07-18.08-gpu]: change \`echo $APPS\` to $APPS

File: jenkins-builds/6.0.UP07-18.08-mc [hunk 2]
```diff
@@ -68,6 +68,7 @@
  jengafettplugin-0.1.eb                             --set-default-module --installpath=$APPS/UES/jenkins/6.0.UP07/mc/easybuild/tools
  likwid-4.3.3-perf_event.eb                         --set-default-module --installpath=$APPS/UES/jenkins/6.0.UP07/mc/easybuild/tools
  line_profiler-2.1-CrayGNU-18.08-python3.eb         --set-default-module --installpath=$APPS/UES/jenkins/6.0.UP07/mc/easybuild/tools
+ malt-1.0.0-CrayGNU-18.08.eb                        --set-default-module --installpath=`echo $APPS`/UES/jenkins/6.0.UP07/mc/easybuild/tools
```
Comment (kraushm) [inline jenkins-builds/6.0.UP07-18.08-mc]: change \`echo $APPS\` to $APPS

**Comment by kraushm** [comment]
retest this please

**Comment by lucamar** [comment]
retest this please

**Comment by lucamar** [comment]
retest this please

**Comment by jgphpc** [comment]
⚠️ do not merge until we fix `production.sh`

**Comment by lucamar** [comment]
The script `production.sh` has been fixed by @teojgo in #987, therefore it should be now safe to merge this pull request.

**Pull request merged**